frontend/src/component/feature/FeatureView/FeatureOverview/ReleasePlan/ReleasePlan.tsx

```diff
@@ -96,6 +100,33 @@ export const ReleasePlan = ({
     const { setToastData, setToastApiError } = useToast();
 
     const [removeOpen, setRemoveOpen] = useState(false);
+    const [
+        planForChangeRequestRemoveDialog,
+        setPlanForChangeRequestRemoveDialog,
+    ] = useState<IReleasePlan | undefined>();
+    const [
+        planForChangeRequestStartMilestoneDialog,
+        setPlanForChangeRequestStartMilestoneDialog,
+    ] = useState<IReleasePlan | undefined>();
+    const [
+        milestoneForChangeRequestDialog,
+        setMilestoneForChangeRequestDialog,
+    ] = useState<IReleasePlanMilestone | undefined>();
```

Comment:
You can use something like `selectedReleasePlan` and share that state across multiple dialogs, since only one of them will be open at a time. I'm also pretty sure you don't need to specify `| undefined` here, it's implicit.

Reply:
What's preventing us from making these dialogs work the same way as the existing `ReleasePlanRemoveDialog`?

Reply:
> since only one of them will be open at a time

We moved the "am I open" part inside the dialog, so this follows that pattern. And to check if it's open - it checks to see if the provided property is defined (or boolean). So if we share, then both delete and start milestone will open in some sequence.

We could move to having an isOpen for them, but then that would be 3 isOpens instead.

Reply:
> but then that would be 3 isOpens instead.

That feels more intuitive for me.